docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,28 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+[Upcoming Feature] For the Soroban RPC, API endpoints will provide JSON format to unpack XDR base64 strings as long as a flag for `xdrFormat: "json"` is passed to the endpoints.
+
+## JSON and XDR Conversion Schema
+
+:::info
+
+SDF has published libraries which support the canonical and round-trippable conversion from XDR to JSON, and from JSON to XDR, without loss of information.
+
+:::
+
+The canonical JSON and XDR Schema is defined by the [rs-stella-xdr library](https://github.com/stellar/rs-stellar-xdr) and by the [@stellar/stellar-xdr-json-web](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web) npm package.
+
+This conversion is visible in the output/input of Stellar CLI, Laboratory View XDR, Soroban RPC, and Hubble events table. The schema provides a round-trippable mean for converting any Stellar XDR type to JSON and converting that JSON back to the identical XDR.
+
+### Key Characteristics of the JSON and XDR Conversion Schema
+
+- Round-Trippable: The JSON format allows for converting from XDR to JSON and back to XDR without loss of information.
+
+- Self-Describing: The JSON format describes itself within the type but does not identify the type that is encoded. This is similar to XDR, which also does not identify the encoded type.
+
+- Single Valid Representation: For any given XDR, there is a single valid JSON representation, and vice-versa. This ensures reproducibility and consistency. However, there are some exceptions.
+
 ## .X files
 
 Data structures in XDR are specified in an interface definition file (IDL). The IDL files used for the Stellar Network are available on [GitHub](https://github.com/stellar/stellar-core/tree/master/src/protocol-curr/xdr).
```

Comment:
Can you update this link while you're at it? I think it should point to https://github.com/stellar/stellar-xdr

Reply:
For sure